db/migrate/20160127210622_migrate_old_configuration_settings.rb

```diff
@@ -58,7 +58,7 @@ def update_old_ui_worker_settings!(config)
     return if roles.include?('user_interface')
 
     roles << 'user_interface'
-    config.store_path(:server, :role, roles.join(','))
+    config.store_path(:server, :role, roles.sort.join(','))
```

Comment:
This should have no impact on anyone who's already run this migration... I'm just sorting these inputs after appending user_interface in case there's already web_services in the list of roles...